zkfacade/src/main/java/com/yahoo/vespa/curator/CuratorCompletionWaiter.java

```diff
@@ -57,8 +57,10 @@ public void awaitCompletion(Duration timeout) {
         List<String> respondents;
         do {
             respondents = curator.framework().getChildren().forPath(barrierPath);
-            log.log(Level.FINE, respondents.size() + "/" + curator.zooKeeperEnsembleCount() + " responded: " +
-                                respondents + ", all participants: " + curator.zooKeeperEnsembleConnectionSpec());
+            if (log.isLoggable(Level.FINE)) {
```

Comment:
You can use `log.log(Level.FINE, () -> `instead

Reply:
Yes, but in this particular case one of the arguments is non-final, and making it final is much more work :)

Reply:
Yes, right, that's a valid case for sure